README.md

```diff
@@ -6,3 +6,11 @@
 [travis-url]: https://travis-ci.org/tox-rs/ws-tox
 
 This server is used to connect JS and tox through WebSocket to create BUI.
+
+## Usage
+
+First you need Rust, libtoxcore and openssl installed. Then you can run ws-tox with
```

Comment:
why openssl? Where is libsodium?

Reply:
Libtoxcore contains libsodium and openssl is required for the websocket library.

Reply:
No if you compile it from source

Reply:
Sure, but we are not going to list all the dependencies of our dependencies, are we?